<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="zwiftModeCheckBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zwift Mode</string>
    </property>
   </widget>
   <widget class="QSlider" name="engineSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>111</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>800</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>900</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="servoPitchSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>181</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>20</number>
    </property>
    <property name="maximum">
     <number>70</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="heartRateTextLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>57</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Heart Rate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="heartRateValueLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>57</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="engineRPMTextLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>82</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Engine speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="engineRPMValueLabel">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>82</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="servoPitchTextLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>149</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fan direction</string>
    </property>
   </widget>
   <widget class="QLabel" name="servoPitchValueLabel">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>149</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="retractButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>220</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Retract</string>
    </property>
   </widget>
   <widget class="QPushButton" name="engineStartButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Engine start</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
